src/wasm-lib/kcl/src/parsing/ast/execute.rs

```diff
@@ -570,6 +542,73 @@ impl Node<CallExpression> {
     }
 }
 
+fn tag_2d_to_3d(result: &mut KclValue, exec_state: &mut ExecState) -> Result<(), KclError> {
```

Comment:
Can we rename this to something more meaningful? The issue is that it does multiple things, including adding tags to program memory (binding output parameters) and updating sketch tags from solids. So I'm not sure what to call it. `handle_stdlib_return()`?